skfda/inference/anova/_anova_oneway.py

```diff
@@ -190,6 +190,7 @@ def _anova_bootstrap(
     random_state: RandomStateLike = None,
     p: int = 2,
     equal_var: bool = True,
+    default_n_features: int = 501,
```

Comment:
Since currently, in case of dealing with a basis representation, the covariance is discretized, it is needed to specify the number of points of evaluation. Here I am using 501 points, same as in `to_grid`.

Reply:
I think we should try to generate the coefficients directly. In any case, you should use a named constant instead of `501`.